config/config_schema.json

```diff
@@ -0,0 +1,171 @@
+{
+  "type": "object",
+
+  "properties": {
+    "DEBUG": {
+      "type": "boolean"
+    },
+    "DEV-MODE": {
+      "type": "boolean"
+    },
+    "PERF": {
+      "type": "boolean"
+    },
+    "SAVE_EVALUATIONS": {
+      "type": "boolean"
+    },
+    "performance-analyser": {
+      "type": "boolean"
+    },
+    "backtesting": {
+      "type": "object",
+      "properties": {
+        "files": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "crypto-currencies": {
+      "type": "object"
```

Comment:
```
{
    "type": "object",
    "patternProperties": {
        "^S_": {
            "type": "object",
            "properties": {
                "pairs": {
                    "type": "array",
                    "uniqueItems": true,
                    "items": {
                        "type": "string"
                }
           }
        }
    }
}
```

https://json-schema.org/understanding-json-schema/reference/array.html
https://json-schema.org/understanding-json-schema/reference/object.html

Reply:
I didn't test it.